### PR TITLE
Merge README and asciidoctor.org content

### PR DESCRIPTION
**Pull request by pepijnve**

### CODE REVIEW

**Comment by mojavelinux** [comment]
GitHub seems to have lost the correlation between my inline comments and the line numbers they reference. Do you need me to repost?

**Comment by pepijnve** [comment]
Not necessary. Comments are still visible at https://github.com/pepijnve/asciidoctor-diagram/commit/06fd1a32a181ac483c9f7d34d4fe52fd83ad21e6#commitcomment-16794913

**Comment by pepijnve** [comment]
@mojavelinux comments integrated. I'm going to delete them from the PR conversation since it's kind of cluttered up now.

**Comment by mojavelinux** [comment]
:+1: 

> Funny that you had so many comments. I copy/pasted the contents from asciidoctor.org so you were reviewing your own work :)

Correct. I was seizing the moment to make improvements ;)

**Comment by pepijnve** [comment]
I've added the missing bits to the README and made some final changes. I'll merge this tomorrow unless I get more feedback.

**Comment by mojavelinux** [comment]
I'll do another review momentarily.

File: README.adoc [hunk 1]
```diff
@@ -1,101 +1,178 @@
...
-=== Additional Requirements
+by first adding the following entry to your project's [.path]_Gemfile_.
 
-Certain diagram types require other tools to be installed seperately.
+.Gemfile
+[source,ruby]
+----
+gem 'asciidoctor-diagram', '~> 1.4.0' // <1>
+----
+<1> Add this line to your application's Gemfile and save the file.
```
Comment (mojavelinux) [inline README.adoc]: I don't think this callout is needed. It says the same thing as the sentence before it and the listing only has one line, so it's hard to confuse.

File: README.adoc [hunk 2]
```diff
@@ -1,101 +1,178 @@
...
+----
+[diagram-type, generated-file-name, generated-image-format] // <1> <2> <3>
+.... // <4>
+Diagram in appropriate syntax
+....
+----
+<1> The first positional attribute in the attribute list specifies the diagram that is being used.
+<2> The second, optional positional attribute assigns a file name (i.e. `target`) to the generated diagram. If a target is not specified an auto-generated name will be used.
+<3> The third, optional positional attribute specifies the image format to create, specified as a three character file extension.
+<4> Place the attribute list directly on top of the delimited literal block (`....`). You can also use an open block as an alternative (`--`).
```
Comment (mojavelinux) [inline README.adoc]: To properly escape the four consecutive dots, they need to be written as:

```
`+....+`
```

File: README.adoc [hunk 3]
```diff
@@ -1,101 +1,178 @@
...
+:uri-plantuml: http://plantuml.sourceforge.net
+:uri-py-plantuml: https://code.google.com/p/asciidoc-plantuml/
+:uri-rackdiag: http://blockdiag.com/en/nwdiag/index.html
+:uri-seqdiag: http://blockdiag.com/en/seqdiag/index.html
+:uri-shaape: https://github.com/christiangoltz/shaape
+:uri-wavedrom: http://wavedrom.com
+:uri-wavedromeditor: https://github.com/wavedrom/wavedrom.github.io/releases
+:uri-wavedromcli: https://github.com/wavedrom/cli
+
+Asciidoctor Diagram is a set of extensions for Asciidoctor that enable you to add diagrams to your AsciiDoc document which you describe using plain text.
```
Comment (mojavelinux) [inline README.adoc]: I think the subject gets mixed up in this opening sentence. Consider rewording it as follows:

```
Asciidoctor Diagram is a set of Asciidoctor extensions that enable you to add diagrams, which you describe using plain text, to your AsciiDoc document.
```

File: README.adoc [hunk 4]
```diff
@@ -1,101 +1,178 @@
...
+The generated file is then inserted into your converted document.
+
+Asciidoctor Diagram was inspired by the {uri-py-plantuml}[AsciiDoc PlantUML filter].
+
+ifdef::status[]
 == Status
 
 image:https://travis-ci.org/asciidoctor/asciidoctor-diagram.svg?branch=master["Linux Build Status", link="https://travis-ci.org/asciidoctor/asciidoctor-diagram"]
 
 image:https://ci.appveyor.com/api/projects/status/4r4gkk5gy3igs6nh/branch/master?svg=true["Windows Build Status", link="https://ci.appveyor.com/project/asciidoctor/asciidoctor-diagram"]
```
Comment (mojavelinux) [inline README.adoc]: These status images might look better as a single line. To accomplish that, simply remove the blank line between them.

As an example of how that looks, see https://github.com/asciidoctor/asciidoctor-gradle-plugin#asciidoctor-gradle-plugin

Comment (mojavelinux) [inline README.adoc]: I also recommend adding the badge for the gem.

```

image:https://img.shields.io/gem/v/asciidoctor-diagram.svg?label=gem%20version[Gem Version, link=https://rubygems.org/gems/asciidoctor-diagram]
```

**Comment by mojavelinux** [comment]
I added a few additional comments. Aside from those modification requests, I think this PR is ready to merge. I'll incorporate the document into asciidoctor.org once that happens.

Looking beyond this update, I think the next step is to introduce a dedicated section for each type of diagram. Users often come to the README looking for a quick example to copy and paste to get their feet wet. Right now, I think you really have to be familiar with a lot of the diagram tools to make use of them.

We could also think of slightly more appropriate examples than the ones we currently have. To get ideas about example diagrams, we can perhaps use http://gravizo.com/#samples as inspiration.

**Pull request merged**